src/tests/fs_tests.rs

```diff
@@ -3100,7 +3100,13 @@ pub mod fs_tests {
         let cage = interface::cagetable_getref(1);
         let path = "";
         // Check for error when directory is empty
-        assert_eq!(cage.mkdir_syscall(path, S_IRWXA), -(Errno::ENOENT as i32));
+        let ret = unsafe {
+            libc::mkdir(path.as_ptr() as *const i8, S_IRWXA)
+        };
```

Comment:
Why we need to check libc `mkdir` here

Reply:
The problem with cage mkdir is when you pass in empty string as directory, it considers the current directory instead of throwing an error EINVAL or something and mkdir passes. 
This test case passes empty string and expects the test case to fail, but it doesn't.